server/controllers/recommendation/reviewPractitionersConcernsController.ts

```diff
@@ -44,7 +44,7 @@ function extractAdditionalLicenceConditions(recommendation: RecommendationDecora
     const { selected, allOptions } = recommendation.cvlLicenceConditionsBreached.additionalLicenceConditions
     return selected.map((s: string) => {
       return {
-        title: allOptions.find((o: Record<string, string>) => o.code === s).text,
+        title: allOptions.find((o: Record<string, string>) => o.code === s)?.text,
```

Comment:
And similar but different here.  Empty string is probably better than undefined.